scripts/ci/cleanup.sh

```diff
@@ -62,22 +62,23 @@ EOF
   systemctl restart orientdb
   systemctl restart lxd
 
-  virsh net-destroy vagrant0
-  virsh net-destroy vagrant-libvirt
-  systemctl restart libvirtd
-
   for vm in dev_dev vagrant_analyzer1 vagrant_agent1 devstack_devstack
   do
     virsh destroy $vm
     virsh undefine $vm
     virsh vol-delete $vm.img default
   done
 
+  virsh net-destroy vagrant0
+  virsh net-destroy vagrant-libvirt
+  systemctl restart libvirtd
+
   rm -rf /tmp/skydive_agent* /tmp/skydive-etcd
   rm -rf /var/lib/jenkins/.vagrant.d/tmp
 
   # time to restart services
   sleep 8
+  ip l del virbr0
```

Comment:
if you are attempting to remove virbr0 should you not use:

```
ip link set virbr0 down
brctl delbr virbr0
```

Also maybe this should be placed after https://github.com/skydive-project/skydive/pull/1707/files#diff-82887418280c1fded8efe3a50691b94dR73

Reply:
Why use `brctl` ? `virbr0` is created by libvirtd so we need to wait a few seconds before deleting it

Reply:
i used this link as reference: https://www.google.co.il/amp/s/www.thegeekdiary.com/how-to-remove-virbr0-and-lxcbr0-interfaces-on-centos-rhel-5-and-rhel-7/amp/